packages/alfa-test-utils/docs/usage/reporting/basic.md

```diff
@@ -0,0 +1,38 @@
+# Reporting on results
+
+See [how to audit a page](../auditing/basic.md) for getting the audit results in the first place.
+
+This page shows basic usage of the scraping, see [common configuration](./configuration.md) for frequent set up such as selecting the rules to run or the issues to show; and [advanced configuration](./advanced.md) for more complex setups.
+
+## Uploading results to the Siteimprove Intelligence Platform
+
+Users of the Siteimprove Intelligence Platform can upload audit results and view them in the Page Report. You first need to [create an API key](https://my2.siteimprove.com/Integrations/Api/ManageApiKeys) in the Siteimprove Intelligence Platform.
+
+Then, simply use the `SIP.upload` function on the audit result, providing the Siteimprove Intelligence Platform credentials. Here, the credentials are passed as environment variables. Depending on setup it might be more convenient to use CI/Cd secrets, … It is not advised to hard-code the credentials in the code as these should be treated as any password.
+
+```typescript
+import { SIP } from "@siteimprove/alfa-test-utils";
+
+const pageReportURL = await SIP.upload(alfaResult, {
+  userName: process.env.SI_USER_NAME!, // email address of the user.
+  apiKey: process.env.SI_API_KEY!, // API key generated in the platform.
+});
+```
+
+This returns a URL to a Siteimprove Intelligence Platform page report showing the audit results, embedded within a [`Result` object](https://github.com/Siteimprove/alfa/blob/main/docs/api/alfa-result.md) to handle errors, use `console.log(pageReportURL.getOrElse(() => pageReportURL.getErrUnsafe()));` to show its value.
+
+> **Note:** By default, when ran from a `git` repository, this will upload some [basic git information](https://github.com/Siteimprove/alfa-integrations/blob/main/docs/api/alfa-test-utils.git.md) if you prefer not to do so, add the `includeGitInfo: false` option to the second argument of `SIP.upload`.
```

Comment:
The link to the git info is currently broken, the rest of the PR is going to fix it 😅